ush/forecast_postdet.sh

```diff
@@ -599,14 +630,22 @@ data_out_GFS() {
           $NCP $file $memdir/RESTART/$file
         done
       fi
-    elif [ $CDUMP = "gfs" ]; then
-      $NCP $DATA/input.nml $ROTDIR/${CDUMP}.${PDY}/${cyc}/atmos/
+    elif [ $CDUMP = "gfs" ] || [ ${RUN} = "gefs" ]; then
```

Comment:
Instead of the nested if, use an additional `elif`.

Reply:
Still think it should be a separate elif. We'll recombine later if necessary (really, I think copying the `input.nml` can just be done for all).

Reply:
agree that copying the input.nml is done for all. If separate it, I should be in the else, because the if conditional use $CDUMP, but for GEFS, it should use $RUN to identify it is for GEFS. What do you think?